packages/core/core/src/createDependency.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import type {DependencyOptions, Dependency, FilePath} from '@parcel/types';
+import md5 from '@parcel/utils/lib/md5';
+
```

Comment:
I think this file got removed

Reply:
Ow yes, thought the merge removed it automatically. Will remove it